backend/src/controllers/project.ts

```diff
@@ -1,31 +1,30 @@
 import { NextFunction, Request, Response } from 'express'
 
-import Project from '../models/project'
+import Project from 'models/project'
 
-export const getProject = ( req: Request, res: Response, next: NextFunction ) => {
+export const getProject = async ( req: Request, res: Response, next: NextFunction ) => {
   const { pid } = req.params
 
   // Retrieve project by id
-  Project.findById(pid)
-    .exec()
-    .then((result) => {
-      return res.status(200).json({
-        automaton: result
-      })
-    }).catch((error) => {
-      return res.status(500).json({
-        message: error.message,
-        error
-      })
+  try {
+    const project = await Project.findById(pid).orFail()
+    return res.status(200).json({
+      project
     })
+  } catch (error) {
+    return res.status(500).json({
```

Comment:
WIll this respond with a `500` if the project doesn't exist? We might want to seperately handle that and return a 404 or remove the `orFail()` call and just return null